src/ui/widget-forms/ContentsQueryForm.js

```diff
@@ -0,0 +1,443 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage, intlShape } from 'react-intl';
+import { Field, FieldArray, reduxForm } from 'redux-form';
+import { Collapse } from 'react-collapse';
+import {
+  Button, Row, Col, FormGroup, Alert,
+} from 'patternfly-react';
+import { maxLength } from '@entando/utils';
+import { isUndefined } from 'lodash';
+
+import RenderTextInput from 'ui/common/form/RenderTextInput';
+import RenderSelectInput from 'ui/common/form/RenderSelectInput';
+import FormLabel from 'ui/common/form/FormLabel';
+import FormSectionTitle from 'ui/common/form/FormSectionTitle';
+import MultiFilterSelectRenderer from 'ui/common/form/MultiFilterSelectRenderer';
+import FiltersSelectRenderer from 'ui/common/form/FiltersSelectRenderer';
+
+const maxLength70 = maxLength(70);
+const CATEGORY_HOME = 'home';
+
+export class ContentsQueryFormBody extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      publishingSettings: false,
+      filters: false,
+      extraOptions: false,
+      frontendFilters: false,
+    };
+  }
+
+  componentDidMount() {
+    const { onDidMount } = this.props;
+    onDidMount();
+  }
+
+  componentDidUpdate(prevProps) {
+    const { selectedContentType: prevContentType } = prevProps;
+    const { selectedContentType, onChangeContentType } = this.props;
+    if (selectedContentType !== prevContentType) {
+      onChangeContentType(selectedContentType);
+    }
+  }
+
+
+  collapseSection(sectionName) {
+    // eslint-disable-next-line react/destructuring-assignment
+    const currentVisibility = this.state[sectionName];
```

Comment:
```suggestion
    const {[sectionName]: currentVisibility} = this.state;
```
this should work and you should be able to remove the eslint-disable comment :)